sources/pug/pages/index.pug

```diff
@@ -259,6 +259,11 @@ block page
   }).section
     #sponsors.container
       h3 Our sponsors
+      .row
+        .col-md-6.offset-md-3
+          a.button.button-primary.box-with-triangle-right.wow.fadeScale( href='/attachments/AsyncAPI-Hackathon-and-Conference-Prospectus.pdf' target="_blank" data-triangle=".button-overlay" )
```

Comment:
```suggestion
          a.button.button-primary.box-with-triangle-right.wow.fadeScale( href='https://conference.asyncapi.com/attachments/AsyncAPI-Hackathon-and-Conference-Prospectus.pdf' target="_blank" data-triangle=".button-overlay" )
```

this is the only way to solve it really, don't know any other solution than some hacks with JS. I think this "hardcode" hack is acceptable if you want it. I do not have a strong opinion here

Reply:
I think it works..  tested in my local machine like below it worked.. 

```
a.button.button-primary.box-with-triangle-right.wow.fadeScale( href='http://localhost:8000/attachments/AsyncAPI-Hackathon-and-Conference-Prospectus.pdf' target="_blank" data-triangle=".button-overlay" )
```